sormas-app/app/src/main/java/de/symeda/sormas/app/immunization/reducedVaccination/VaccinationReducedListAdapter.java

```diff
@@ -30,7 +30,7 @@ public void onBindViewHolder(@NonNull RecyclerView.ViewHolder holder, int positi
 			pageHolder.setOnListItemClickListener(this.mOnListItemClickListener);
 
 			Date vaccinationDate = pageHolder.binding.getData().getVaccinationDate();
-			pageHolder.binding.setVariable(BR.fontColorGrey, vaccinationDate != null && vaccinationGrayoutDate.before(vaccinationDate));
+			pageHolder.binding.setVariable(BR.fontColorGrey, vaccinationDate == null || (vaccinationDate != null && vaccinationGrayoutDate.before(vaccinationDate)));
```

Comment:
I believe the != null check is no longer needed now because it's already covered by the null check before